code/lib/core-server/src/utils/get-server-channel.ts

```diff
@@ -47,6 +42,21 @@ export class ServerChannelTransport {
       .filter((c) => c.readyState === WebSocket.OPEN)
       .forEach((client) => client.send(data));
   }
+
+  private handleEvent(raw: WebSocket.RawData) {
+    const data = raw.toString();
+    const event =
+      typeof data === 'string' && isJSON(data)
+        ? parse(data, { allowFunction: false, allowClass: false })
+        : data;
+
+    const pageString = colors.blue(this.page);
+    const eventString = colors.green(event.type);
+    const message = `${pageString} received ${eventString} (${data.length})`;
+    logger.verbose(`${message} ${event.args.map(inspect).join(' ')}`);
```

Comment:
I'm not sure what's going on with this line, it seems different to the other `handleEvent` functions (that call `pretty.debug(...event.args)`. I don't know where I got it from exactly (and it is causing tests to fail).

Feel free to change as you see fit @ndelangen